superset/assets/javascripts/explore/stores/controls.jsx

```diff
@@ -1197,6 +1197,16 @@ export const controls = {
     description: t('What should be shown on the label?'),
   },
 
+  sql: {
+    type: 'TextAreaControl',
+    label: t('Code'),
+    description: t('Put your sql here. This feature is for developers who understand superset. If you are not sure, please do not fill in anything, it will be ignored.'),
```

Comment:
It this is a joke i don't think it's funny

Reply:
I have used and verified this feature in a production environment because I can't find any way to call a stored procedure. This feature does solve my problem.